src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -32,8 +33,26 @@ import {
 			(click)="onClick($event)"
 			[disabled]="disabled"
 			[attr.title]="title">
-			<ng-content></ng-content>
+			<ng-container *ngTemplateOutlet="tempOutlet"></ng-container>
 		</button>
+
+		<a
+			*ngIf="href !== undefined"
```

Comment:
could we just do a regular `*ngIf="href"` and `*ngIf="!href"` instead of explicitly checking against undefined? We should catch empty strings and other falsey values with the implicit check